lighthouse-treemap/app/src/main.js

```diff
@@ -377,7 +391,7 @@ class TreemapViewer {
       if (!dataRow.unusedBytes) return '';
 
       const percent = Math.floor(100 * dataRow.unusedBytes / dataRow.resourceBytes);
-      return `${percent}% bytes unused`;
+      return `${Util.i18n.formatNumber(percent)}% bytes unused`;
```

Comment:
I don't think we have a good way to handle this; thoughts? It's just a tooltip...

should we do the "nearly correct" thing and just string concat this formatted number w/ a Util.i18n.bytesUnused ?

Reply:
but how would we do the value replacement at runtime? can we do something like

```
const data = replaceIcuMessages({string: str_(myStringWithPlaceholders, {value: 123}})
// get string from data ....
```

Reply:
Hope

Reply:
Yeah IDK, but either way I think we can punt for now :)

Maybe we start an issue with i18n frontend use cases? This happens in the report to some extent too we just chopped off the explanations to make them less helpful.